
From: Zwane Mwaikambo <zwane@linuxpower.ca>

The 'noapic' kernel parameter only disables IOAPIC use and not all the
APICs (which would include local APICs) in the system.

Signed-off-by: Zwane Mwaikambo <zwane@fsmlabs.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/Documentation/kernel-parameters.txt |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN Documentation/kernel-parameters.txt~update-noapic-description Documentation/kernel-parameters.txt
--- 25/Documentation/kernel-parameters.txt~update-noapic-description	2004-10-03 16:43:56.063808560 -0700
+++ 25-akpm/Documentation/kernel-parameters.txt	2004-10-03 16:43:56.067807952 -0700
@@ -756,8 +756,8 @@ running once the system is up.
 
 	noalign		[KNL,ARM] 
  
-	noapic		[SMP,APIC] Tells the kernel not to make use of any
-			APIC that may be present on the system.
+	noapic		[SMP,APIC] Tells the kernel to not make use of any
+			IOAPICs that may be present in the system.
 
 	noasync		[HW,M68K] Disables async and sync negotiation for
 			all devices.
_
